image/templates/helm/stackrox-secured-cluster/internal/expandables.yaml.htpl

```diff
@@ -42,3 +42,22 @@ scanner:
   dbServiceTLS:
     cert: true
     key: true
+[<- if .FeatureFlags.ROX_SCANNER_V4 >]
+scannerV4:
+  indexer:
+    nodeSelector: true
```

Comment:
just noticed this is "expandable" here but not for Central. How come?

Reply:
I don't know how that was lost. I have added into central-services as well for consistency reasons. This expansion mechanism is a fancy feature of our Helm chart allowing the user to inject contents from files into Helm input values. This is in particular useful for things like passwords, but the charts also support this kind expansion for other values.